lib/project_types/theme/cli.rb

```diff
@@ -18,6 +20,7 @@ module Tasks
 
   module Forms
     autoload :Create, Project.project_filepath('forms/create')
+    autoload :Serve, Project.project_filepath('serve')
```

Comment:
Is this actually being used? It doesn't seem like we have a form for `serve`.

Reply:
Good catch, this is a remnant of some testing I did!